runtime.md

```diff
@@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
 See the specification of the `config.json` file for the definition of these fields.
 The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
 In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
-
-## Hooks
-
-Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
```

Comment:
What about post stop? If we don't pin the namespaces, I cannot extract files from the mount namespaces after the init process dies, for example.